src/main/java/lukemccon/airdrop/packages/CreatePackageGui.java

```diff
@@ -0,0 +1,146 @@
+package lukemccon.airdrop.packages;
+
+import lukemccon.airdrop.Airdrop;
+import lukemccon.airdrop.helpers.ChatHandler;
+import lukemccon.airdrop.helpers.PermissionsHelper;
+import org.bukkit.Bukkit;
+import org.bukkit.ChatColor;
+import org.bukkit.Material;
+import org.bukkit.entity.HumanEntity;
+import org.bukkit.entity.Player;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.inventory.InventoryClickEvent;
+import org.bukkit.event.inventory.InventoryDragEvent;
+import org.bukkit.inventory.Inventory;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.inventory.meta.ItemMeta;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class CreatePackageGui implements Listener {
+    private final Inventory inv;
+    private final int ROW_SIZE = 9;
+    private final int SAVE_CANCEL_BACK_PADDING = 4;
+    private Package pkg;
+    private final String name;
+    private final double price;
+    private static final String[] controlItemNames = { "Save", "Cancel", "Back" };
+
+    public CreatePackageGui(String name, double price) {
+
+        this.name = name;
+        this.price = price;
+
+        int inventorySize = 27;
+        int packageCount = PackageManager.getNumberofPackages();
+
+        // Logic to determine how large to make the inventory
+        inventorySize = (int) (ROW_SIZE * (Math.ceil(((packageCount + SAVE_CANCEL_BACK_PADDING)/ROW_SIZE) + 1 ) + 1));
```

Comment:
On line 36 inventorySize is initialized to 27, then it gets rewritten here. It should probably just be initialized here.